../gegl-0.4.30/operations/external/tiff-load.c:99:9: warning:
implicit declaration of function 'strptime' is invalid in C99

Index: operations/external/tiff-load.c
--- operations/external/tiff-load.c.orig
+++ operations/external/tiff-load.c
@@ -18,7 +18,7 @@
 
 #include "config.h"
 #ifdef HAVE_STRPTIME
-#define _XOPEN_SOURCE
+#define _XOPEN_SOURCE 700
 #include <time.h>
 #endif
 #include <glib/gi18n-lib.h>
